internal/app/triggerfactory.go

```diff
@@ -0,0 +1,121 @@
+//
```

Comment:
This is not a new file. GIT isn't recognizing it was moved and renamed.

Reply:
Fix Technocrats back to Technotects in this one if you get a chance please 😂